openhantek/src/hantekdso/dsocontrol.h

```diff
@@ -108,7 +108,7 @@ class DsoControl : public DsoCommandQueue {
 
     /// \brief The resulting tuple of the computeBestSamplerate() function
     struct BestSamplerateResult {
-        unsigned downsampler = 0;
```

Comment:
Either ceil or floor is applied to the downsampler result depending on the model, so we need to have floating point value here.